nestkernel/free_layer.h

```diff
@@ -234,9 +229,25 @@ FreeLayer< D >::set_status( const DictionaryDatum& d )
   }
   else
   {
-    this->extent_ = max_point - this->lower_left_;
-    this->extent_ += epsilon * 2;
-    this->lower_left_ -= epsilon;
+    if ( this->node_collection_->size() <= 1 )
+    {
+      throw KernelException( "If only one node is created, 'extent' must be specified." );
+    }
+
+    const auto positional_extent = max_point - this->lower_left_;
+    const auto center = ( max_point + this->lower_left_ ) / 2;
+    Position< D > rounded_center;
+    for ( int d = 0; d < D; ++d )
+    {
+      // Round to the first decimal.
+      rounded_center[ d ] = ( std::round( 10.0 * center[ d ] ) ) / 10;
```

Comment:
Why this rounding? I would leave just the center that we get from the positions. If the user wants a specific center, they can specify the extent.